### ISSUE TITLE
A11y_SQL Azure Data Studio_Notebooks -Python Dependencies Wizard_Keyboard: Keyboard focus is not clearly visible on the buttons('Next', 'Cancel').

### ISSUE THREAD

**Issue by SachinChavhan317** [microsoft/azuredatastudio#15967]
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

GitHubTags: #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Win32_Jun2021;#SQL Azure Data Studio;#Win10;#DesktopApp;#WCAG2.4.7;#FocusVisible;

Environment Details:
Application Name: Azure Data Studio
Version: 1.30.0-insider (user setup)
Windows Version: Windows10

Repro Steps:

1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Right-click connection and click New Notebook
4. Open the Command Palette by pressing CTRL+SHIFT+P (Windows) or CMD+SHIFT+P (Mac). Alternatively, you can select the gear icon at the bottom left of the screen and select “Command Palette…” from the menu that appears.
5. Type “Python” into the text box that appears. Select “Configure Python for Notebooks” from the menu. Python Dependencies Wizard will appear.
6. Provide any Input file details & Next.
7. Click Install.
7. Check Keyboard focus is clearly visible on the buttons('Next', 'Cancel') or not.

Actual:
Keyboard focus is not clearly visible on the buttons('Next', 'Cancel').

Expected:
Keyboard focus should be clearly visible on the buttons('Next', 'Cancel').

User Impact:
low vision users will face difficulty while accessing the Buttons('Next', 'Cancel').

**Comment by Charles-Gagnon** [comment]
@SachinChavhan317 Please make sure you're using the latest versions of insiders when doing testing, which should be **1.31.0-insider**

There was a recent bunch of fixes brought in, one of which was adding a highlight border to focused buttons. 

![image](https://user-images.githubusercontent.com/28519865/124156805-eb996380-da4c-11eb-852c-692179809915.png)

![image](https://user-images.githubusercontent.com/28519865/124156845-f48a3500-da4c-11eb-872d-5df119240541.png)

This should fix the issue so I'm going to close this out now, let us know if there's something I'm missing.

**Comment by Harika1702** [comment]
<img width="166" alt="image" src="https://user-images.githubusercontent.com/100270074/217489888-2532a280-193d-4f3a-b3da-91caa0c015a0.png">
<img width="190" alt="image" src="https://user-images.githubusercontent.com/100270074/217490041-3f6334a7-b06e-4226-87f3-68a2c2db2ec2.png">
Verified this issue in the below environment.
Application name: Azure data studio
Version: 1.40.2 (user setup)
Commit: 661384637db384fe5d4e6224069adbe708580b16
Date: 2022-12-22T22:17:02.658Z
VS Code: 1.67.0
Electron: 17.4.11
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.14-electron.0
OS: Windows_NT x64 10.0.22623
GitHub Tags: #Closed;
